paymentsheet-example/src/androidTest/java/com/stripe/android/lpm/TestSunbit.kt

```diff
@@ -0,0 +1,37 @@
+package com.stripe.android.lpm
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.stripe.android.BasePlaygroundTest
+import com.stripe.android.paymentsheet.example.playground.settings.Country
+import com.stripe.android.paymentsheet.example.playground.settings.CountrySettingsDefinition
+import com.stripe.android.paymentsheet.example.playground.settings.Currency
+import com.stripe.android.paymentsheet.example.playground.settings.CurrencySettingsDefinition
+import com.stripe.android.test.core.TestParameters
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+internal class TestSunbit : BasePlaygroundTest() {
+    private val testParameters = TestParameters.create(
+        paymentMethodCode = "sunbit",
+    ) { settings ->
+        settings[CountrySettingsDefinition] = Country.US
+        settings[CurrencySettingsDefinition] = Currency.USD
+    }
+
+    @Ignore("Complex authorization handling required")
```

Comment:
Does this need to be ignored? Using this annotation makes it so the test doesn't run

Reply:
This is safe to enable! Just removed this line